lib/util/lowercase-keys.ts

```diff
@@ -0,0 +1,10 @@
+export = function lowercaseKeys (object?: { [key: string]: any}) {
```

Comment:
I’d prefer to split this up into lines for readability

```suggestion
export = lowercaseKeys

function lowercaseKeys (object?: { [key: string]: any}) {
```